### ISSUE TITLE
objectType changing from decimal to string

### ISSUE THREAD

**Issue by JetskiOK** [robincornelius/libedssharp#146]
Hello,
I noticed editor changing objectType  from standard numeric to string when saving even unmodified XML file. 

"7" -> "VAR" 
"8" -> "ARRAY" 
"9" -> "REC"

Not sure what the reason behind it, but it would be nice to fix this issue. Many CanOpen tools having problems with reading such format.

**Comment by robincornelius** [comment]
Thanks for that, I'll get this fixed and a new release is overdue.
XDD is not quite as tested as some of the other bits so i'm happy others are using it and feeding back issues.

**Comment by JetskiOK** [comment]
Hello,
thanks for checking. 
Yes we used zip for v0.6-xdd-alpha-81-gb562769. Version below it doesn't have xdd import/export functionality. File name libedssharp-0.6-xdd-2.zip. I am attaching pdf copy of edited file. Search for //ADW to see what I changed.

Just want express our thanks you for publishing and maintaining his project. This software is our life saver!
[CanOpenXDD.pdf](https://github.com/robincornelius/libedssharp/files/2581130/CanOpenXDD.pdf)
